tests/random_test.py

```diff
@@ -1032,6 +1032,38 @@ def testLogistic(self, dtype):
     for samples in [uncompiled_samples, compiled_samples]:
       self._CheckKolmogorovSmirnovCDF(samples, scipy.stats.logistic().cdf)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_n={}_shape={}_dtype={}"\
```

Comment:
For better test names, we usually use `jtu.format_shape_dtype_string`; see e.g. https://github.com/google/jax/blob/ab37701ce3e6973ea7be1d549ae4d0d7ca9ee826/tests/lax_numpy_test.py#L979